pom.xml

```diff
@@ -394,14 +399,43 @@
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-all</artifactId>
-        <version>4.1.28.Final</version>
+        <version>4.1.42.Final</version>
```

Comment:
We can extract the netty version altogether and put it to one place.

Reply:
thanks i changed